CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
-# Change Log
+# Changelog
 All notable changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](http://semver.org/).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Is this here intentionally? There doesn't seem to be anything "unreleased".

Reply:
Yep, its a permanent new section in the changelog we should keep https://keepachangelog.com/en/1.0.0/#effort